plugin/evm/limit_order.go

```diff
@@ -349,36 +352,77 @@ func (lop *limitOrderProcesser) runMatchingTimer() {
 }
 
 func (lop *limitOrderProcesser) loadMemoryDBSnapshot() (acceptedBlockNumber uint64, err error) {
+	acceptedBlockNumber, err = lop.loadMemoryDBSnapshotFromFile()
+	if err != nil || acceptedBlockNumber == 0 {
+		acceptedBlockNumber, err = lop.loadMemoryDBSnapshotFromHubbleDB()
+	}
+	return acceptedBlockNumber, err
+}
+
+func (lop *limitOrderProcesser) loadMemoryDBSnapshotFromHubbleDB() (uint64, error) {
```

Comment:
[todo, non-blocker] remove this function and remaining hubbledb code and raise a work in progress PR right now only